jaas/command/src/main/java/org/apache/karaf/jaas/command/ManageRealmCommand.java

```diff
@@ -35,14 +35,14 @@
 @Service
 public class ManageRealmCommand extends JaasCommandSupport {
 
-    @Option(name = "--realm", description = "JAAS Realm", required = false, multiValued = false)
+    @Option(name = "--realm", description = "JAAS Realm. Must be combined with --module", required = false, multiValued = false)
     @Completion(RealmCompleter.class)
     String realmName;
 
     @Option(name = "--index", description = "Realm Index", required = false, multiValued = false)
```

Comment:
I would add the "must be combined" statement here and leave realm option description as it is.